### PR TITLE
editoast: make liveness probe configurable

### PR DESCRIPTION
**Pull request by bloussou**

### CODE REVIEW

**Review by ElysaSrc — APPROVED** [review]
LGTM !

File: values.yaml [hunk 1]
```diff
@@ -37,6 +37,10 @@ editoast:
     type: ClusterIP
     port: 80
     targetPort: 80
+  livenessProbe:
+    initialDelaySeconds: 5
```
Comment (flomonster) [inline values.yaml:41]: Today this value is set to `0` (default). I think we can keep it that way.

**Review by flomonster — APPROVED** [review]
LGTM

**Review by younesschrifi — APPROVED** [review]
LGTM

**Pull request merged**